docs/hub/ollama.md

```diff
@@ -85,6 +87,14 @@ You can optionally configure a system prompt by putting it into a new file named
 
 To change sampling parameters, create a file named `params` in the repository. The file must be in JSON format. For the list of all available parameters, please refer to [this documentation](https://github.com/ollama/ollama/blob/main/docs/modelfile.md#parameter).
 
+## Run Private GGUFs from the Hugging Face Hub
+
+You can run private GGUFs from your personal account or from an associated organisation account in three simple steps:
+
+1. Copy your Ollama SSH key you can do so via: `cat ~/.ollama/id_ed25519.pub | pbcopy`
+2. Add the corresponding key to your Hugging Face account by going to [your account settings](https://huggingface.co/settings/keys) and clicking on `Add new key`.
+3. On terminal, run `huggingface-cli login` to authenticate with your Hugging Face account.
+4. That's it! You can now run private GGUFs from the Hugging Face Hub: `Ollama run hf.co/{username}/{repository}`.
```

Comment:
```suggestion
4. That's it! You can now run private GGUFs from the Hugging Face Hub: `ollama run hf.co/{username}/{repository}`.
```

Reply:
this works w/ `ollama` too no?

Reply:
ah no, I'm blind, I have it the other way, fixed.